src/GraphQL.Harness/GraphQL.Harness.csproj

```diff
@@ -5,6 +5,8 @@
     <NoWarn>$(NoWarn);1591</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'GraphiQL' " />
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'Playground' " />
```

Comment:
What is it ?

Reply:
I am using visual studio for mac. It adds these lines automatically for some reason. I will remove them.